benchmark/realnest/02_list_sort_text.benchmark

```diff
@@ -0,0 +1,17 @@
+# name: benchmark/realnest/02_list_sort_text.benchmark
+# description: list_sort text entries
+# group: [realnest]
+
+name list_sort_text
+group real_nest
+
+require json
+
+require httpfs
+
+cache real_nest.duckdb
+
+load benchmark/realnest/load.sql
+
+run
+SELECT list_sort(list_transform(body_text, x -> x.text)) FROM cord;
```

Comment:
`list_sort(body_text)`, if the runtime does not explode, which it should not.

Reply:
resolved